app/src/main/java/com/keylesspalace/tusky/db/AccountEntity.kt

```diff
@@ -41,6 +41,7 @@ data class AccountEntity(
     var accountId: String = "",
     var username: String = "",
     var displayName: String = "",
+    var createdAt: String = "",
```

Comment:
But where is this ever displayed to the user

Reply:
I am not sure, are you suggesting we don't store the date in the DB at all then? (Sorry, I'm new here so I have no idea what I'm doing)

Reply:
The `com.keylesspalace.tusky.db.AccountEntitiy` is for storing the users logged into the app. I don't see why we would need to store their account creation date. We only want to show in on the profiles, where data is alwasy fetched from the api (and not cached), so it is enough to have it in `com.keylesspalace.tusky.entity.Account`.

Reply:
Gotcha, i'll remove all the DB modifications. It makes sense.

Reply:
Reverted.